tracer/src/Datadog.Trace/Util/RandomNumberGenerator.cs

```diff
@@ -0,0 +1,76 @@
+// <copyright file="RandomNumberGenerator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// https://github.com/dotnet/runtime/blob/main/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/RandomNumberGenerator.cs
+// https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Random.Xoshiro256StarStarImpl.cs
+// https://prng.di.unimi.it/xoshiro256starstar.c
```

Comment:
I wonder if we should vendor this properly, so we can update it easily? It should be easy to update obviously (copy-paste a couple of methods), but when will we ever do that 😅 On second thoughts, vendoring is probably much to much like hard work...

Reply:
Can we vendor single files?

Reply:
> Can we vendor single files

Not currently AFAIK. so not worth the effort I think